### PR TITLE
Fix document new form

### PR DESCRIPTION
**Pull request by mattlindsey**
Fix submit button on document new form.

### CODE REVIEW

**Comment by nickthecook** [comment]
Hey, what exactly is being fixed here?

**Comment by mattlindsey** [comment]
Submit button doesn't work (at least for me) I think because the f.submit line is outside for the begin/end for the form_for.  This fixes that.

**Comment by nickthecook** [comment]
Ah, okay. They both work for me, so I'll merge this.

File: app/views/documents/new.html.erb [hunk 1]
```diff
@@ -51,7 +50,8 @@
               <div class="flex-1"></div>
               <%= cp.text_field :overlap, value: default_chunk_overlap(default_chunking_method), size: 10, maxlength: 10, class: "rounded-md dark:bg-secondary-925 text-right" %>
             </div>
-          <% end %>
+            <% end%>
```
Comment (nickthecook) [inline app/views/documents/new.html.erb:53]:
```suggestion
            <% end %>
```

**Comment by nickthecook** [comment]
Odd, the `f.submit` is not outside the form, just outside the `f.fields_for`. Any idea why this changes behaviour for you? Anything in your JS console when the button doesn't work?

File: app/views/documents/new.html.erb (hunk 1 above)
Comment (nickthecook) [reply app/views/documents/new.html.erb:53]: Looks like some indentation might need an update too.

Comment (mattlindsey) [reply app/views/documents/new.html.erb:53]: In the original code the <% end %> doesn't match up with the form_for. I think rubocop and/or standard.rb can check erb files.  Rubocop does find some other minor fixes currently too.

**Comment by mattlindsey** [comment]
> Odd, the `f.submit` is not outside the form, just outside the `f.fields_for`. Any idea why this changes behaviour for you? Anything in your JS console when the button doesn't work?

The <% end %> doesn't match up the the form_for in the original code.  I think rubocop or rubocop with 'standard' gem will check erb files if you want.  Also running 'rubocop' shows a bunch of other minor fixes, but nothing major.

**Comment by nickthecook** [comment]
Looks like you've moved all the inputs into the rounded border.

The rounded border is intended to be around to upload field and link field, so it was clear the "OR" only applies to those two fields.

**Comment by mattlindsey** [comment]
> Looks like you've moved all the inputs into the rounded border.
> 
> The rounded border is intended to be around to upload field and link field, so it was clear the "OR" only applies to those two fields.

I don't think so.  The border is the same

> Looks like you've moved all the inputs into the rounded border.
> 
> The rounded border is intended to be around to upload field and link field, so it was clear the "OR" only applies to those two fields.

Ah. I see.  The problem was just the <div> and fields_for were reversed, and that only caused a problem on my browser for some odd reason.

**Pull request merged**